server/src/main/java/org/jboss/as/server/ServerService.java

```diff
@@ -424,10 +424,18 @@ public void undeploy(DeploymentUnit context) {
             Notification notification = new Notification(ModelDescriptionConstants.BOOT_COMPLETE_NOTIFICATION, PathAddress.pathAddress(PathElement.pathElement(CORE_SERVICE, MANAGEMENT),
                     PathElement.pathElement(SERVICE, MANAGEMENT_OPERATIONS)), ServerLogger.AS_ROOT_LOGGER.bootComplete());
             getNotificationSupport().emit(notification);
-            bootstrapListener.printBootStatistics();
+            List<String> configFiles = new ArrayList<String>();
```

Comment:
Minor detail, you can use Java Type inference capabilities, so instead of  `new ArrayList<String>()` use `new ArrayList<>()` here.